test/e2e/pages/vl-accordion.page.js

```diff
@@ -35,6 +31,11 @@ class VlAccordionPage extends Page {
         return this._openAccordion('#accordion-1');
     }
 
+    async getStandaardAccordionTextContent() {
```

Comment:
Dit soort functies kunnen best vermeden worden, de logica verplaatsen naar het webcomponent page object zodat alles rechtstreeks in de test uitgevoerd kan worden. Ik zal het aanpassen.